vlingo-eventjournal/src/main/java/io/vlingo/eventjournal/counter/events/CounterIncreasedEvent.java

```diff
@@ -0,0 +1,22 @@
+package io.vlingo.eventjournal.counter.events;
+
+import com.google.gson.Gson;
+import io.vlingo.symbio.Event;
+import io.vlingo.symbio.Metadata;
+import io.vlingo.symbio.State;
+
+import java.util.UUID;
+
+public class CounterIncreasedEvent {
```

Comment:
It's best not to name events with the word `Event` in them because that word is generally not part of the ubiquitous language. So `CounterIncreased` is a better name.